internal/pkg/cli/init.go

```diff
@@ -109,6 +110,7 @@ func newInitOpts(vars initVars) (*initOpts, error) {
 	spin := termprogress.NewSpinner(log.DiagnosticWriter)
 	id := identity.New(defaultSess)
 	deployer := cloudformation.New(defaultSess, cloudformation.WithProgressTracker(os.Stderr))
+	iam := iam.New(defaultSess)
```

Comment:
Can we avoid using the package name as the var name? for both init and app init 🙏